### DISCUSSION TITLE
Search for Exact Matches by Using Quotation Marks in the Browser

### DISCUSSION THREAD

**Discussion by SchofieldPriest** [BlafKing/sd-civitai-browser-plus#112] [Ideas]
If you try to enclose a search term in quotation marks (" ") on CivitAI, results will only show for models that match the search term exactly (case-insensitive). Could this kind of search filter be implemented in a future update?

**Comment by BlafKing** [comment]
Unfortunately this functionality relies on [CivitAI's Public REST API](https://github.com/civitai/civitai/wiki/REST-API-Reference) which is responsible for returning the search results, so I won't be able to add this feature myself.

The CivitAI site uses a different version of their API which is private and not open to the public, so that's why this feature is only available on the site.

If you'd like, you can post a feature request on the CivitAI github instead [here](https://github.com/orgs/civitai/discussions), since they should be able to implement this, instead of me :)

**Comment by SchofieldPriest** [comment]
@BlafKing, how would I rephrase a feature request like this in the official CivitAI github? "How can I have an extension search CivitAI for exact search results by using quotation marks? Is this site search functionality currently private?"

**Reply by BlafKing** [reply]
I'd post something like this:

Title:
Search for Exact Matches by Using Quotation Marks in public REST API

Message:
Currently the query parameter returns all models containing the provided search term, leading to broad results.
I'd like to request a "search term" format by using quotation marks to allow users to find models with exact search term matches.
It looks like this is already supported on the CivitAI website, so I'd love to see it in the Public REST API as well!

Thanks!

**Reply by SchofieldPriest** [reply]
Just made a post! I'll see what they respond with.
